tm-monitor/monitor/monitor.go

```diff
@@ -231,3 +231,31 @@ func (m *Monitor) updateNumValidatorLoop() {
 		}
 	}
 }
+
+
+//--> types
+
+//type NetworkAndNodes struct {
+//	Network *Network `json:"network"`
+//	Nodes   []*Node  `json:"nodes"`
+//}
+
+type NetworkAndNodes struct {
+	NetStatus *NetworkStatus `json:"network_status"`
+	NodeStatus []*NodeStatus `json:"node_status"`
+}
+
+type NetworkStatus struct{
```

Comment:
Not sure I understand why we need these additional structs. Could you explain why we can't just serialise/deserialise `*monitor.Network` and `[]*monitor.Node`?

Reply:
@melekes  I got errors `panic serving [::1]:50855: interface conversion: string is not error: missing method Error` with serialization of *monitor.Network &[]*monitor.Node
Also, I think we should keep `status` information simple.
More details can be shown in `status\network`& `status/node`

Reply:
https://github.com/tendermint/go-amino#floating-points

that's because go-amino does not support floating types. we just need to add field tag `amino:"unsafe"` to floating types

Reply:
sorry for late reply